phpBB/phpbb/install/console/command/install/config/default_config.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\install\console\command\install\config;
+
+use phpbb\language\language;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class default_config extends \phpbb\console\command\command
+{
+	/**
+	 * @var language
+	 */
+	protected $language;
+
+	/**
+	 * Constructor
+	 *
+	 * @param language $language
+	 */
+	public function __construct(language $language)
+	{
+		$this->language = $language;
+
+		parent::__construct(new \phpbb\user($language, 'datetime'));
+	}
+
+	/**
+	 *
+	 * {@inheritdoc}
+	 */
+	protected function configure()
+	{
+		$this
+			->setName('install:config:default')
```

Comment:
I wonder if we really need this command.
I think it can be removed (it should be part of the documentation)